src/common/bitfield.h

```diff
@@ -13,7 +13,12 @@
 #include <string>
 #include <vector>
 
-#include "span.h"
+#if defined(__CUDACC__)
+#include <thrust/copy.h>
+#include <thrust/device_ptr.h>
+#endif  // defined(__CUDACC__)
+
+#include "xgboost/common/span.h"
```

Comment:
Sometimes you use `<>` for XGBoost headers and sometimes you use `""`. Any rationale for this?

Reply:
@hcho3 I use `""` in internal code and `<>` in public headers.  The difference becomes important when you export an interface.  With proper configuration in CMake, headers in system path are included with compiler flag `-isystem` instead of `-I`.  Using `""` will fail.

Reply:
Interesting. So `<>` will cause the compiler to look into system paths.